packages/cozy-device-helper/src/flagship.ts

```diff
@@ -0,0 +1,22 @@
+import { Platform } from 'react-native'
```

Comment:
@acezard We imprt from react-native but I don't see it in devDep. Don't we have webpack errors now ? 

Can't we import from type/react-native or something else?